src/compo/mopitt_co_nc2ioda.py

```diff
@@ -230,10 +239,18 @@ def main():
         help="path of IODA output file",
         type=str, required=True)
 
+    optional = parser.add_argument_group(title='optional arguments')
+    optional.add_argument(
+        '-r', '--date_range',
```

Comment:
nit picking, but this includes 'hour', so I would probably call it a 'time range'. But I won't block this from merging because of it.

Reply:
yeah sorry i really don't have time for this level of detail right now because of the release deadline that we have. i'll make a note and change this on the next PR on iodaconv for sure.

Reply:
done now